packages/story-editor/src/components/library/panes/text/textPane.js

```diff
@@ -74,10 +105,30 @@ function TextPane(props) {
           disabled
         />
       )}
-
-      <Section
-        title={__('Presets', 'web-stories')}
-        onPointerOver={() => enableSmartTextColor && generateCanvasFromPage()}
+      <SmartColorToggle>
+        <Text
+          as="label"
+          htmlFor={toggleId}
+          size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+        >
+          {__('Adaptive text colors', 'web-stories')}
+        </Text>
+        <Tooltip
+          title={__(
+            'May impact performance when inserting text',
+            'web-stories'
+          )}
+        >
+          <Toggle
+            id={toggleId}
+            name={toggleId}
```

Comment:
What's the name for? If the input isn't inside a form, the name doesn't really serve a purpose.